src/Login/Login.test.js

```
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Login } from '../index';
+import { configureKeycloak } from '../keycloak';
+
+describe('Login', () => {
+  beforeAll(() => {
+    configureKeycloak();
```

I believe that calling `configureKeycloak`without any arguments should throw an error.